.github/pull_request_template_v2.md

```
@@ -0,0 +1,128 @@
+# Context
+
+### Spec PR
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)